lib/zxcvbn.rb

```diff
@@ -24,25 +24,33 @@ class Error < StandardError; end
   )
 
   def self.zxcvbn(password, user_inputs = [])
-    start = (Time.now.to_f * 1000).to_i
-    matches = Matching.omnimatch(password, user_inputs)
-    result = Scoring.most_guessable_match_sequence(password, matches)
-    result["calc_time"] = (Time.now.to_f * 1000).to_i - start
-    attack_times = TimeEstimates.estimate_attack_times(result["guesses"])
-    attack_times.each do |prop, val|
-      result[prop] = val
-    end
-    result["feedback"] = Feedback.get_feedback(result["score"], result["sequence"])
-    result
+    Tester.new.zxcvbn(password, user_inputs)
   end
 
   def self.test(password, user_inputs = [])
-    Result.new(Zxcvbn.zxcvbn(password, user_inputs))
+    Tester.new.test(password, user_inputs)
   end
```

Comment:
What about making both of these use a shared, memoized Tester instance? That'd still lazy load the first time, but avoid reloading all the time.

To unload / reload on every use, an app could just instantiate a new Tester each time.